ui/src/app/views/admin-user-bypass/component.tsx

```diff
@@ -30,11 +30,11 @@ export class AdminUserBypass extends React.Component<
     super(props);
     const {profile} = props;
     const initialModules = [
-      !!profile.betaAccessBypassTime ? AccessModule.BETAACCESS : null,
-      !!profile.complianceTrainingBypassTime ? AccessModule.COMPLIANCETRAINING : null,
-      !!profile.dataUseAgreementBypassTime ? AccessModule.DATAUSEAGREEMENT : null,
-      !!profile.eraCommonsBypassTime ? AccessModule.ERACOMMONS : null,
-      !!profile.twoFactorAuthBypassTime ? AccessModule.TWOFACTORAUTH : null,
+      ...(!!profile.betaAccessBypassTime ? [AccessModule.BETAACCESS] : []),
```

Comment:
`!!` is redundant here. You're already in a context where `profile.betaAccessBypassTime` is a predicate and will therefore be automatically coerced to a boolean; explicit coercion is not necessary.